yaki_database/create_tables.sql

```diff
@@ -0,0 +1,189 @@
+
+-- CREATE USER IDs
+
+CREATE SEQUENCE IF NOT EXISTS public.user_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
+    CACHE 1;
+
+ALTER SEQUENCE public.user_id_seq
+    OWNER TO yaki;
+
+
+-- CREATE CAPTAIN IDs
+
+CREATE SEQUENCE IF NOT EXISTS public.captain_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
+    CACHE 1;
+
+ALTER SEQUENCE public.captain_id_seq
+    OWNER TO yaki;
+
+
+-- CREATE TEAM IDs
+
+CREATE SEQUENCE IF NOT EXISTS public.team_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
+    CACHE 1;
+
+ALTER SEQUENCE public.team_id_seq
+    OWNER TO yaki;
+
+
+-- CREATE TEAM MATE IDs
+
+CREATE SEQUENCE IF NOT EXISTS public.team_mate_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
+    CACHE 1;
+
+ALTER SEQUENCE public.team_mate_id_seq
+    OWNER TO yaki;  
+
+
+-- CREATE LOCATION IDs
+
+CREATE SEQUENCE IF NOT EXISTS public.location_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
+    CACHE 1;
+
+ALTER SEQUENCE public.location_id_seq
+    OWNER TO yaki;
+
+
+-- CREATE DECLARATION IDs
+
+CREATE SEQUENCE IF NOT EXISTS public.declaration_id_seq
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    MAXVALUE 9223372036854775807
+    CACHE 1;
+
+ALTER SEQUENCE public.declaration_id_seq
+    OWNER TO yaki;
+
+-- Then create the databases' main tables for captains, team, team mates, locations and declarations
+
+-- CREATE TABLE FOR USERS
+
+CREATE TABLE IF NOT EXISTS public."user"
+(
+    user_id integer NOT NULL DEFAULT nextval('user_id_seq'::regclass),
+    user_last_name character varying(100) COLLATE pg_catalog."default",
+    user_first_name character varying(100) COLLATE pg_catalog."default",
+    user_email character varying(100) COLLATE pg_catalog."default",
+    user_login character varying(100) COLLATE pg_catalog."default",
+    user_password character varying(255) COLLATE pg_catalog."default",
+    CONSTRAINT user_pkey PRIMARY KEY (user_id)
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."user"
+    OWNER to yaki;
+
+
+--- CREATE TABLE FOR TEAM MATES
+
+CREATE TABLE IF NOT EXISTS public.team_mate
+(
+    team_mate_id integer NOT NULL DEFAULT nextval('team_mate_id_seq'::regclass),
+    team_mate_team_id integer NOT NULL,
+    team_mate_user_id integer,
+    CONSTRAINT "TEAM_MATE_pkey" PRIMARY KEY (team_mate_id),
+    CONSTRAINT team_mate_team_id_fkey FOREIGN KEY (team_mate_team_id)
+        REFERENCES public.team (team_id) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public.team_mate
+    OWNER to yaki;
+
+
+-- CREATE TABLE FOR CAPTAINS
+
+CREATE TABLE IF NOT EXISTS public.captain
+(
+    captain_id integer NOT NULL DEFAULT nextval('captain_id_seq'::regclass),
+    captain_team_id integer,
```

Comment:
what's the point of this column ? I don't really understand

Reply:
We thought it was necessary to store the id of the team in which the captain is in charge.But now that i'm thinking about it, it's unnecessary because we already store the captain's id inside the team table. I will remove it for the next PR.